apps/antalmanac/src/components/RightPane/CoursePane/CourseRenderPane.tsx

```diff
@@ -113,9 +116,12 @@ const RecruitmentBanner = () => {
  */
 const SectionTableWrapped = (
     index: number,
-    data: { scheduleNames: string[]; courseData: (WebsocSchool | WebsocDepartment | AACourse)[] }
+    data: { 
+        scheduleNames: string[]; 
+        courseData: (WebsocSchool | WebsocDepartment | AACourse)[] 
+        course: AACourse; }
```

Comment:
fmt: semicolon is missing on one line, and the closing curly brace for `data` should be on a newline.